docs-v2/components/ArcadeEmbed.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const ArcadeEmbed = ({
+  src, title,
+}) => {
+  const embedUrl = `${src}`;
+
+  return (
+    <div style={{
```

Comment:
You should always use `className="relative ..."` so you can use the Tailwind classes instead of passing a custom `style`.

Reply:
You mean instead of defining custom styles altogether? So figure out how to translate those custom style defs to corresponding tailwind classes? FWIW I copied those styles directly from the embed code that app Arcade spits out: 
```
<div style="position: relative; padding-bottom: calc(56.25% + 41px); height: 0; width: 100%;"><iframe src="https://demo.arcade.software/hljzHunRakVX30QipJ1y?embed&show_copy_link=true" title="Connect using custom OAuth client" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"></iframe></div>
```